spyder_unittest/widgets/tests/test_unittestgui.py

```diff
@@ -224,6 +224,28 @@ def test_run_tests_using_unittest_and_display_results(qtbot, tmpdir,
     assert model.index(1, 1).data(Qt.ToolTipRole) == 'test_foo.MyTest.test_ok'
     assert model.index(1, 2).data(Qt.DisplayRole) == ''
 
+@pytest.mark.parametrize('framework', ['unittest', 'pytest', 'nose'])
+def test_run_with_no_tests_discovered_and_display_results(qtbot, tmpdir,
+                                                          monkeypatch, framework):
+    """Basic integration test."""
+    os.chdir(tmpdir.strpath)
+    testfilename = tmpdir.join('test_foo.py').strpath
```

Comment:
I think you can remove this line.

Reply:
Yes, you're right. I overlooked this one. Thanks.